core/src/main/java/org/verapdf/pdfa/validation/profiles/Profiles.java

```diff
@@ -311,25 +311,20 @@ public static Rule ruleFromValues(final RuleId id, final String object, final St
 	/**
 	 * Returns a {@link Rule} instance initialised with the passed values.
 	 *
-	 * @param id
-	 *            the {@link RuleId} id for the {@link Rule}
-	 * @param object
-	 *            a String that identifies the Object that the rule applies to
-	 * @param deferred
-	 *            a Boolean that identifies the deferred property of the rule
-	 * @param description
-	 *            a textual description of the {@link Rule}.
-	 * @param test
-	 *            a JavaScript expression that is the test carried out on a
-	 *            model instance
-	 * @param error
-	 *            the {@link ErrorDetails} associated with the{@link Rule}.
-	 * @param references
-	 *            a list of further {@link Reference}s for this rule
+	 * @param id          the {@link RuleId} id for the {@link Rule}
+	 * @param object      a String that identifies the Object that the rule applies to
+	 * @param deferred    a Boolean that identifies the deferred property of the rule
+	 * @param tags		  a String that identifies the tag that the rule applies to
```

Comment:
a String that contains comma-separated list of tags associated with this rule

Reply:
fixed